FluentScheduler/Extensions/DateTimeExtensions.cs

```diff
@@ -90,6 +90,45 @@ public static DateTime Next(this DateTime current, DayOfWeek dayOfWeek)
 			return current.AddDays(offsetDays);
 		}
 
+    /// <summary>
+    /// Nexts the Nth weekday.
+    /// </summary>
+    /// <param name="d">The day.</param>
+    /// <param name="n">The number of weekdays to advance.</param>
+    /// <returns>Advanced date.</returns>
+    public static DateTime NextNWeekday(this DateTime d, int n)
```

Comment:
Can you make the variable names be a bit more descriptive than single characters?